python/lsst/pipe/tasks/selectImages.py

```diff
@@ -515,7 +516,7 @@ def runDataRef(self, dataRef, coordList, makeDataRefList=True,
                 continue
             if self.config.minPsfFwhm and sizeFwhm < self.config.minPsfFwhm:
                 continue
-            psfSizes.append(psfSize)
+            psfSizes.append(sizeFwhm)
```

Comment:
Fair enough. I would like the gen3 one to have a clearer name more like `fwhmPsfSize`, and I agree consistency is helpful, so yes please do change the variable name throughout. I've been bitten more than once by assuming the wrong thing about what PSF size I'm working with.

Reply:
You know what. Now that we have `BestSeeingSelectVisits` etc, `BestSeeingWcsSelectImagesTask` will NEVER be run in Gen3 mode. Selecting 5 of our best seeing 6 calexps makes no sense.  Instead of changing the variable names, what do you say I just delete this whole `run` method?

Reply:
Yes, I thought the old selector was gen2 only; the only reason to keep any of it around is for gen2 use. If the run method you speak of is intended for gen3, then delete away.